wiki/Skinning/Compendium/en.md

```diff
@@ -1,373 +1,10 @@
-- [Xi-Style](/community/forums/topics/4833) by [Xiao](/users/17894)![][o!s]
-- [Xi-Style G2](/community/forums/topics/59787) by [Xiao](/users/17894)![][o!s]![][o!t]![][o!m]
-
-## Y
-
-- [Yet Another Miku Skin](/community/forums/topics/450498) by [rkgk](/users/7620476)![][o!s]
-- [YoRHa](/community/forums/topics/540757) by [Choilicious](/users/2129634)![][o!s]
-- [YUGEN](/community/forums/topics/365036) by [Garin](/users/2130664)![][o!s]![][o!t]![][o!c]![][o!m]
-- [Yui](/community/forums/topics/651827) by [dlwnstns99](/users/4585661)![][o!s]
-- [Yukino](/community/forums/topics/651827) by [dlwnstns99](/users/4585661)![][o!s]
-- [Yuuki](/community/forums/topics/651827) by [dlwnstns99](/users/4585661)![][o!s]
-- [Yuzuki Yukari](/community/forums/topics/342349) by [Elzapatoverde](/users/3717733)![][o!s]![][o!c]
-
-## Z
-
-- [Zero Two](/community/forums/topics/651827) by [dlwnstns99](/users/4585661)![][o!s]
-- [zglow](/community/forums/topics/641141) by [Saikkimus](/users/9820272)![][o!s]![][o!t]![][o!c]![][o!m]
-
-## Others
 
-- ["Guest Tracks" Summary & ViPs 01](/community/forums/topics/598236) by [atturbo555](/users/5730417)![][o!s]
-- [-cherryglow-](/community/forums/topics/446984) by [Falbere](/users/6531756)![][o!s]![][o!m]
-- [- V O I D -](/community/forums/topics/402100) by [Ben_5917](/users/6026593)![][o!s]![][o!t]![][o!c]![][o!m]
-- [- [ The Font is Sans ] -](/community/forums/topics/616189) by [h3oCharles](/users/8712289)![][o!s]![][o!t]![][o!c]![][o!m]
-- [--[cyber.KIN3TIK]--](/community/forums/topics/602283) by [badwolfvi](/users/8997109)![][o!s]
-- [01 Vocaloid](/community/forums/topics/558470) by [HalfPinter](/users/8377034)![][o!s]
-- [20XX](/community/forums/topics/696128) by [Hidah](/users/10804561)![][o!s]
-- [4Rhythm](/community/forums/topics/639393) by [atturbo555](/users/5730417)![][o!s]
-- [8ghts](/community/forums/topics/727680) by [NeemaTori](/users/11829195)![][o!s]![][o!t]![][o!c]
-- [[Nightcore HD]](/community/forums/topics/135588) by [evil_genius](/users/2526988)![][o!s]![][o!t]![][o!c]![][o!m]
-- [»multiberry«](/community/forums/topics/607808) by [atturbo555](/users/5730417)![][o!s]
-- [Ōkami - 大](/community/forums/topics/275293) by [Elzapatoverde](/users/3717733)![][o!s]
-- [▰ shocked! ▰](/community/forums/topics/248517) by [Angelo](/users/2094431)![][o!s]
-- [♥ loveHeart](/community/forums/topics/359459) by [Rebecca](/users/6509768)![][o!s]
-- [✿ 空母ヲ級 (Aircraft carrier Wo-Class)](/community/forums/topics/478380) by [Choilicious](/users/2129634)![][o!s]
-- [東方Project - Hata no Kokoro](/community/forums/topics/224323) by [Elzapatoverde](/users/3717733)![][o!s]![][o!t]![][o!c]![][o!m]
-- [東方Project - Hinanawi Tenshi](/community/forums/topics/211868) by [Elzapatoverde](/users/3717733)![][o!s]![][o!t]![][o!c]![][o!m]
-- [東方Project - Kirisame Marisa](/community/forums/topics/530165) by [Elzapatoverde](/users/3717733)![][o!s]![][o!c]
+Compendium of all completed osu! skins, splitted into the following categories:
```

Comment:
Compendium of all completed osu! skins. They are currently splitted into the following categories: